src/structures/interfaces/InteractionResponses.js

```diff
@@ -224,6 +225,21 @@ class InteractionResponses {
     return options.fetchReply ? this.fetchReply() : undefined;
   }
 
+  /**
+   * Presents a modal component
+   * @param {Modal|ModalOptions} modal The modal to present
+   * @returns {Promise<void>}
+   */
+  async presentModal(modal) {
```

Comment:
`showModal` maybe?

I'm not sure what the implications of `present` and `show` here really are, but neither are amazing, if I had to choose I'd go with "showing" something, instead of "presenting" something.

Reply:
Yeah Siris and I discussed this and we both liked show. Will update tomorrow